config/flags_directory.go

```diff
@@ -62,7 +61,7 @@ func RegisterFlagsDirectory(flags *[]cli.Flag) error {
 
 	FlagConfigDir.Value = filepath.Join(currentDir, "config")
 	FlagDataDir.Value = filepath.Join(userHomeDir, ".mysterium")
-	FlagLogDir.Value = FlagDataDir.Value
+	FlagLogDir.Value = filepath.Join(FlagDataDir.Value, "logs")
```

Comment:
Also update cli docs mentioning that default path would be data dir + logs.

Reply:
Actually, it's kind of mentioned, because we set flag's `Value`. Here's an example (`node --help`):
```
--log-dir value                      Log directory for storing log files (default: "/Users/tadas/.mysterium/logs")
```